shared/test_run_query.go

```diff
@@ -296,14 +332,17 @@ func contains(s []string, x string) bool {
 
 // Loads any keys for a revision prefix or full string match
 func loadKeysForRevision(store Datastore, query Query, sha string) (result TestRunIDs, err error) {
```

Comment:
Shall we rename this to `loadIDsForRevision`, and `loadKeysForBrowserVersion` to `loadIDsForBrowserVersion`?

By the way, `loadKeysForBrowserVersion` currently returns a mapset, which is inconsistent with `loadKeysForRevision`. Looking at the call sites, we should perhaps make both return mapsets.

Reply:
Done.